### PR TITLE
Update table

### PR DESCRIPTION
**Pull request by daniel-sargeant**
### Description (optional)

Update Table to match current frontend version so it works when we consume from component-library.

### CODE REVIEW

**Comment by chan-fai-leung** [comment]
It's weird we export a table component but doesn't have a story to associate it with. @daniel-sargeant do you think you can add a story to demonstrate how the table should look in storybook. 

I would recommend to create a story/task for this kind of work

**Comment by chan-fai-leung** [comment]
Code LGTM

**Comment by daniel-sargeant** [comment]
> I would recommend to create a story/task for this kind of work

@chan-fai-leung I created a story stub for now and can create a ticket to complete it – should that ticket be in Asana or Jira?

**Comment by chan-fai-leung** [comment]
> > I would recommend to create a story/task for this kind of work
> 
> @chan-fai-leung I created a story stub for now and can create a ticket to complete it – should that ticket be in Asana or Jira?

jira

**Comment by daniel-sargeant** [comment]
https://submittable.atlassian.net/browse/SUB-7854

**Comment by chan-fai-leung** [comment]
@daniel-sargeant do you think you can add a story for table so it can be capture in chromatic for any deltas

**Comment by daniel-sargeant** [comment]
> @daniel-sargeant do you think you can add a story for table so it can be capture in chromatic for any deltas

I added one earlier, does it need anything added?

**Comment by chan-fai-leung** [comment]
> > @daniel-sargeant do you think you can add a story for table so it can be capture in chromatic for any deltas
> 
> I added one earlier, does it need anything added?

Can you add some sample data to it. I see an empty table

**Comment by daniel-sargeant** [comment]
> Can you add some sample data to it. I see an empty table

Sure, updated.

**Review by chan-fai-leung — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 # production
 /build
+/storybook-static
```
Comment (chan-fai-leung) [inline .gitignore:16]: can you also update .npmignore too. Dont want to publish this to the consumer

Comment (chan-fai-leung) [reply .gitignore:16]: you ran build-storybook, in our CI/CD we do not run that script so storybook-static will not be generated and publish

**Pull request merged**